### DISCUSSION TITLE
13/05/24

### DISCUSSION THREAD

**Discussion by Pamela198713** [enbalde-ispc/enbalde-ispc#141] [Daily Sprint]
Qué hice ayer
Qué hago hoy
Impedimentos

**Comment by Pamela198713** [comment]
Ayer hice parte del trabajo. 
Hoy veré qué falta y completaré.
Comunicación en en grupo.

**Comment by rpgrca** [comment]
Ayer actualicé el script de inicialización de base de datos, también hice algunos ajustes a la implementación del botón de pago. También armé un script de prueba de registración de usuario en Postman como ejemplo y toqué un poco de documentación. Hoy veré si continuo con lo de Android para el segundo sprint.

**Comment by LuciaL152** [comment]
Ayer estuve terminando un trabajo pero no es referido al ispc y hoy estuve terminando de diseñar los prototipos de la app en figma.

**Comment by fmdcba** [comment]
Ayer trabajé en la bitácora del ieee para el proyecto, hoy estuve con los casos de prueba y en un rato voy a pasar lo de emprendedurismo a formato ppt para su presentación
